scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -211,7 +211,7 @@
         title="Overlapping gene panels"
         data-bs-content="{% for panel in matching_panels %}
           {{ panel.panel_name|safe }}<br>
-        {% endfor %}"
+        {% else %} No ovelapping gene panels {% endfor %}"
```

Comment:
` {% else %}`? But what if are you closing? It looks slightly different for the both usages?